packages/postgres-database/src/simcore_postgres_database/utils.py

```diff
@@ -81,3 +81,11 @@ def as_postgres_sql_query_str(statement) -> str:
         dialect=postgresql.dialect(),  # type: ignore[misc]
     )
     return f"{compiled}"
+
+
+def assemble_array_groups(user_group_ids: list[int]) -> str:
```

Comment:
MINOR: `utils` should only have generic utils . 
I would transform this function into a generic free function that transforms an iterable of something that can be stringified into an SQL array

```python

def to_sql_array_str(items: Iterable) -> str :
    items = ', '.join(f"'{item}'" for item in items)
    return ( f"array[{items}]"  if items else "array[]::text[]"
    )
```